src/AccountManager.mo

```diff
@@ -471,15 +477,15 @@ module {
     };
 
     /// Processes the transfer of funds for withdrawal.
-    func processWithdrawTransfer(to : ICRC1.Account, amount : Nat) : async* {
+    func processWithdrawTransfer(from_subaccount : ?Blob, to : ICRC1.Account, amount : Nat) : async* {
```

Comment:
Not sure if this function should really be used for both steps of the transfer. It compares the amount against `minimum(#withdrawal)` which isn't really needed in the second step of the transfer (though it should be ok, just redundant in most cases). But maybe there are edge cases where the admin increases the minimum in the middle of on ongoing withdrawal, but the withdrawal was initiated when the minimum was lower. Then this would break that withdrawal.